config/nimbus.yaml

```diff
@@ -21,6 +21,3 @@ features:
       - channel: production
         value: { "something": "wicked" }
 
-types:
-  objects: {}
-  enums: {}
```

Comment:
@rhelmer The docs show a different syntax [for enums](https://experimenter.info/fml-spec/#enumerations) and [for objects](https://experimenter.info/fml-spec/#object-types). Do you know which of these is outdated? I followed the docs for now.

Reply:
I suspect the docs are right, thanks.